ballerina-tests/tests/61_any_type_pb.bal

```diff
@@ -287,6 +294,6 @@ public type Person2 record {|
 const string ROOT_DESCRIPTOR_61_ANY_TYPE = "0A1136315F616E795F747970652E70726F746F1A19676F6F676C652F70726F746F6275662F616E792E70726F746F22310A07506572736F6E3112120A046E616D6518012001280952046E616D6512120A04636F64651802200128055204636F646522430A07506572736F6E3212120A046E616D6518012001280952046E616D6512120A04636F64651802200128055204636F646512100A03616464180320012809520361646432A1030A0D416E7954797065536572766572123A0A0A756E61727943616C6C3112142E676F6F676C652E70726F746F6275662E416E791A142E676F6F676C652E70726F746F6275662E416E792200123A0A0A756E61727943616C6C3212142E676F6F676C652E70726F746F6275662E416E791A142E676F6F676C652E70726F746F6275662E416E792200123A0A0A756E61727943616C6C3312142E676F6F676C652E70726F746F6275662E416E791A142E676F6F676C652E70726F746F6275662E416E79220012450A1373657276657253747265616D696E6743616C6C12142E676F6F676C652E70726F746F6275662E416E791A142E676F6F676C652E70726F746F6275662E416E792200300112450A13636C69656E7453747265616D696E6743616C6C12142E676F6F676C652E70726F746F6275662E416E791A142E676F6F676C652E70726F746F6275662E416E7922002801124E0A1A6269646972656374696F6E616C53747265616D696E6743616C6C12142E676F6F676C652E70726F746F6275662E416E791A142E676F6F676C652E70726F746F6275662E416E79220028013001620670726F746F33";
 
 public isolated function getDescriptorMap61AnyType() returns map<string> {
```

Comment:
can we remove this function from generating?

Reply:
This will be removed with https://github.com/ballerina-platform/ballerina-standard-library/issues/2797